### DISCUSSION TITLE
The "Extension" icon functions in different manner on different toolbars

### DISCUSSION THREAD

**Discussion by GHNewbiee** [Floorp-Projects/Floorp#1074] [Show and tell]
Hi,

One may assume function duplication and remove one icon from one toolbar. Please consider to adapt a 2nd icon slightly modified from the main one.

Remark: Same valid for any other icon which may behave different from time to time.

*Tia*
![floorp](https://github.com/Floorp-Projects/Floorp/assets/26875662/2f70b0be-c6d0-4a41-aab5-aca1547291cf)

**Comment by breadbeforerice** [comment]
That is indeed true. The shortcut was there first and the icon above came to Firefox later. 
If there are others, it would be easier to write them down.
